astropy_healpix/core.py

```diff
@@ -77,10 +84,92 @@ def level_to_nside(level):
     nside : int
         The number of pixels on the side of one of the 12 'top-level' HEALPix tiles.
     """
+    level = np.asarray(level, dtype=np.int64)
+
     _validate_level(level)
     return 2 ** level
 
 
+def nside_to_level(nside):
+    """
+    Find the HEALPix level for a given nside. (this is given by log2(nside))
```

Comment:
The first summary line of docstrings should be short and sweet.
Remove the `(...)` part or move it to a later line, like:
```
Find the HEALPix level for a given nside.

This is given by ``level = log2(nside)``.
```